app/components/apply-page/apply-client.tsx

```diff
@@ -0,0 +1,59 @@
+const ApplyClient = () => {
+  return (
+    <div className="p-8 max-w-3xl mx-auto">
+      <div className="pb-8">
+        <h1 className="text-2xl font-semibold text-indigo-600 pb-2">Clients</h1>
+        <p>
+          Are you a nonprofit organization in the greater Boston area? If you
+          would like further information about Code4Community or wish to discuss
+          a project with us, drop us an email by filling out the form below!
+        </p>
+      </div>
+      <div className="flex flex-row gap-4 pb-8">
+        <div className="w-full">
+          <label htmlFor="nameInput" className="font-semibold block pb-1">
+            Name*
+          </label>
+          <input
+            id="nameInput"
+            name="nameInput"
+            defaultValue="Your Name"
```

Comment:
nit: let's use placeholder instead of default value so that the user doesn't need to delete our default value. Also as a followup we can take advantage of https://ui.shadcn.com/docs/components/input for the inputs